database/dataGeneration.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const faker = require('faker');
+const youtubeURL = require('./youtube');
+const constants = require('./constants');
+
+faker.seed(3604035591);
+faker.local = 'en_US';
+
+const generateImages = () => {
+  let numOfImages = faker.random.number({ min: 6, max: 20 });
+  const images = [];
+  while (numOfImages > 0) {
+    const s3File = faker.random.number({ min: 1, max: 1000 });
+    const imageName = `image${s3File}.jpg`;
+    images.push(imageName);
+    numOfImages -= 1;
+  }
+  return images;
+};
+
+const generateVideos = () => {
+  let numOfVideos = faker.random.number({ min: 1, max: 5 });
+  const videos = [];
+  while (numOfVideos > 0) {
+    const url = faker.random.arrayElement(youtubeURL);
+    videos.push(url);
+    numOfVideos -= 1;
+  }
+  return videos;
+};
+
+const generateTags = () => {
+  let numOfTags = 4;
+  const tags = [];
+  while (numOfTags > 0) {
+    const tempTags = faker.random.arrayElement(constants.allTags);
+    tags.push(tempTags);
+    numOfTags -= 1;
+  }
+  return tags;
+};
+
+const reviews = [
+  'Overhwelmingly Positive',
+  'Very Positive',
+  'Positive',
+  'Mostly Positive',
+  'Mixed',
+  'Mostly Negative',
+  'Negative',
+  'Very Negative',
+  'Overwhelmingly Negative',
+];
+
+const writeGameRecords = fs.createWriteStream('gameData.csv');
+writeGameRecords.write('images,videos,tags,name,description,all_reviews,recent_reviews,release_date,developer,publisher\n', 'utf8');
```

Comment:
Hey just a heads up here: when you go to seed your Postgres db, you might want to comment out this line because it will probably interpret this first line as an actual row to enter instead of the headers. I'm not sure if there is a way to skip the header line when loading from csv, but it could be worth checking out.

Reply:
Awesome, thanks for the tip!